src/main/java/nova/internal/core/depmodules/FakeLanguageModule.java

```diff
@@ -18,12 +18,15 @@
  * along with NOVA.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-package nova.wrappertests.depmodules;
+package nova.internal.core.depmodules;
```

Comment:
Why did this move package? If a wrapper doesn't implement a LanguageManager it should copy it's own `FakeLanguageManager`. There's no precedent for moving fakes into core.